rules/xcodeproj.bzl

```diff
@@ -269,9 +278,22 @@ def _xcodeproj_impl(ctx):
             framework_search_paths.append("\"%s\"" % fi)
         target_settings["FRAMEWORK_SEARCH_PATHS"] = " ".join(framework_search_paths)
         target_settings["GCC_PREPROCESSOR_DEFINITIONS"] = " ".join(["\"%s\"" % d for d in target_info.cc_defines.to_list()])
+
         if target_info.product_type == "application":
             target_settings["INFOPLIST_FILE"] = "$BAZEL_STUBS_DIR/Info-stub.plist"
             target_settings["PRODUCT_BUNDLE_IDENTIFIER"] = target_info.bundle_id
+        else:
```

Comment:
Why do we only do this if the target is not an ios-application? Shouldn't we always do it if the `target_info.swift_defines` are there?

Reply:
nvm i found that for normal application it will show up. the one looked at is test host